app/models/server/models/WebdavAccounts.js

```diff
@@ -7,7 +7,7 @@ export class WebdavAccounts extends Base {
 	constructor() {
 		super('webdav_accounts');
 
-		this.tryEnsureIndex({ user_id: 1, server_url: 1, username: 1, name: 1 }, { unique: 1 });
+		this.tryEnsureIndex({ user_id: 1, server_url: 1, name: 1 }, { unique: 1 });
```

Comment:
if you want to change an index, you'll have to create a migration script to remove the old one. otherwise when a running server upgrades to the new version it will have both indices, which might lead to unexpected results

Reply:
@sampaiodiego Thank you for the review. I'll make the required changes in a day or two, though I'm not quite sure how to go about this migration script one. Currently, I'm modifying another PR and then I'll do this :)

Reply:
I'd like to first understand, why you're changing this index?

Reply:
Well, I changed it because `username` was redundant if `user_id` is already there. But on second thoughts I think I can keep this as it is keeping in mind it's so much trouble.

Reply:
I see. but that `username` is not even the Rocket.Chat's username, it is actually the webdav account username, so it makes even less sense =)

I'll ask you then to remove the index completely using a migration, you can see how [migration 122 did](https://github.com/RocketChat/Rocket.Chat/blob/5ef81257efd2b2f951e2fbd3a54922d116e33619/server/startup/migrations/v122.js#L7) .. and on the model add a new index, only with `user_id` field (not `unique`) for query performance reasons. does it makes sense? thx